commit b78585fe8d9d519364d44d923bd4696c1487134a
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Sun Jun 23 21:51:56 2013 +0300

    Enable exceptions for the EXR thumbnailer.
    
    The OpenEXR headers can throw exceptions, and clang complains when
    exceptions are not enabled (kdelibs disables them by default).

diff --git a/kioslave/thumbnail/CMakeLists.txt b/kioslave/thumbnail/CMakeLists.txt
index 46005e0..8e5692f 100644
--- kioslave/thumbnail/CMakeLists.txt
+++ kioslave/thumbnail/CMakeLists.txt
@@ -110,6 +110,10 @@ if(OPENEXR_FOUND)
 
    target_link_libraries(exrthumbnail ${KDE4_KIO_LIBS} ${OPENEXR_LIBRARIES})
 
+   # OpenEXR headers use exceptions; at least clang refuses to build the target
+   # when exceptions are not enabled.
+   set_target_properties(exrthumbnail PROPERTIES COMPILE_FLAGS "${KDE4_ENABLE_EXCEPTIONS}")
+
    install(TARGETS exrthumbnail DESTINATION ${PLUGIN_INSTALL_DIR})
    install( FILES exrthumbnail.desktop DESTINATION ${SERVICES_INSTALL_DIR})
 
